src/pages/VolunteersJobListingFS.page

```diff
@@ -156,11 +158,17 @@
             <input type="button" value="{!$Label.labelButtonSignUp}"  class="cssJobShiftSignup btn" onclick="OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '', '{!JSENCODE(job.Name)}', ''); return false;"/>
         </apex:outputPanel>
         
-        <apex:outputPanel id="panelExternalJob" rendered="{!job.External_Signup_Url__c != null}" >
+        <apex:outputPanel id="panelExternalJob" rendered="{!job.External_Signup_Url__c != null && NOT(eventAllShiftsInThePast)}" >
         	<apex:outputLink target="_blank" styleClass="externalJobLink" value="{!job.External_Signup_Url__c}" >{!$Label.labelExternalSignupUrl}</apex:outputLink>
         	<br /><br />
         </apex:outputPanel>
-            
+
+        <apex:outputPanel id="panelEventInThePast" rendered="{!eventAllShiftsInThePast}" >
+        	<apex:outputText value="{!$Label.labelEventInThePast}" />
+        	<apex:outputText rendered="{!job.First_Shift__c != null}" value=" First Shift: " />
```

Comment:
"First Shift:" needs to be a custom label so it can be localized or overridden.  thanks.